### PR TITLE
Update plan-for-teams-live-events.md

### PR DESCRIPTION
**Pull request by ashwinma14**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-live-events/plan-for-teams-live-events.md [hunk 1]
```diff
@@ -59,13 +59,13 @@ The user must have:
...
-|Guest users                   |  No<sup>1</sup> |  No                  |
-|Everyone in federated company |  No<sup>2</sup> |  No                  |
+|Guest users                   |  Yes            |  No                  |
+|Everyone in federated company |  Yes<sup>1</sup>|  No                  |
 |Everyone in company           |  Yes            |  Yes                 |
 |Specific groups / people      |  Yes            |  Yes                 |
 
-<sup>1</sup> Can watch live events if the live event is set up using the **Org-wide** option.<br>
-<sup>2</sup> Can only watch live events as anonymous users.
+<sup>1</sup> Federated Attendees can only be invited through People & Group <br>
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/plan-for-teams-live-events.md]:
```suggestion
<sup>1</sup> Federated attendees can only be invited through People and Group. <br>
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@vinbelms please review. LMK if it's OK to merge.

**Comment by LanaChin** [comment]
fyi @JohanFreelancer9, I updated the text with your copy edit suggestion. Merged.

**Pull request merged**